autoPyTorch/utils/backend.py

```diff
@@ -205,12 +205,12 @@ def temporary_directory(self) -> str:
 
     def _make_internals_directory(self) -> None:
         try:
-            os.makedirs(self.internals_directory)
+            os.makedirs(self.internals_directory, exist_ok=True)
         except Exception as e:
             if self._logger is not None:
                 self._logger.debug("_make_internals_directory: %s" % e)
         try:
-            os.makedirs(self.get_runs_directory())
+            os.makedirs(self.get_runs_directory(), exist_ok=True)
```

Comment:
so, the main idea is to run post hoc ensemble fitting. In case the user wants to fit an ensemble using the results from a previous search, they can simply pass the old directory and we can fit an ensemble over that. @nabenabe0928 the automl_common is being used in the `development` branch. Here we are still using the old code.